shared-bindings/synthio/BlockBiquad.c

```diff
@@ -0,0 +1,156 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: Copyright (c) 2023 Jeff Epler for Adafruit Industries
+//
+// SPDX-License-Identifier: MIT
+
+#include "py/enum.h"
+#include "py/objproperty.h"
+#include "py/runtime.h"
+#include "shared-bindings/synthio/BlockBiquad.h"
+#include "shared-bindings/util.h"
+
+//| class FilterMode:
+//|     """The type of filter"""
+//|
+//|     LOW_PASS: FilterMode
+//|     """A low-pass filter"""
+//|     HIGH_PASS: FilterMode
+//|     """A high-pass filter"""
+//|     BAND_PASS: FilterMode
+//|     """A band-pass filter"""
+//|
```

Comment:
Notch is defined below but not here? Is that on purpose?

Reply:
should be added here, good catch!